### PR TITLE
docs: add travis ci secret environment variables data leak

### PR DESCRIPTION
**Pull request by lirantal**
Adding the September 2021 secrets environment variables leak security incident for any publicly forked repositories during a week period.

### CODE REVIEW

File: supply-chain-security/compromises/2021/travis-ci.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+* Any environment variables that were used as protected secrets in the project's
+configuration were leaked and available to CI runs in publicly forked repositories.
+
+## Remediation
+
+* Travis CI notified users and customers to rotate their keys and secrets
+
+## Type of Compromise
+
+Environment variables.
```
Comment (lumjjb) [inline supply-chain-security/compromises/2021/travis-ci.md:24]: Compromise type should be one of https://github.com/cncf/tag-security/blob/main/supply-chain-security/compromises/compromise-definitions.md, and you can add free from text elaborating on the env variables

File: supply-chain-security/compromises/2021/travis-ci.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+service. On September 23rd, 2021, they reported [1] that users and customers
+of their service, who had sensitive information in public repositories, and
+of which these were forked, had leaked these secrets in the build
+infrastructure logs were the CI service was ran.
+
+
+## Impact
+
+* All public repositories that observed forks during the period of September 3rd
+and September 10th
```
Comment (lumjjb) [inline supply-chain-security/compromises/2021/travis-ci.md:14]:
```suggestion
and September 10th 2021
```

File: supply-chain-security/compromises/2021/travis-ci.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+configuration were leaked and available to CI runs in publicly forked repositories.
+
+## Remediation
+
+* Travis CI notified users and customers to rotate their keys and secrets
+
+## Type of Compromise
+
+Environment variables.
+## References
```
Comment (lumjjb) [inline supply-chain-security/compromises/2021/travis-ci.md:25]: Linter

supply-chain-security/compromises/2021/travis-ci.md:25 MD022/blanks-around-headings/blanks-around-headers Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Above] [Context: "## References"]
```suggestion

## References
```

File: supply-chain-security/compromises/2021/travis-ci.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Secrets leak and other sensitive information due to a bug in Travis CI
```
Comment (lumjjb) [inline supply-chain-security/compromises/2021/travis-ci.md:1]:
```suggestion
# Secrets leak and other sensitive information due to a bug in Travis CI
```

linter

supply-chain-security/compromises/2021/travis-ci.md:1:73 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]

File: supply-chain-security/compromises/2021/travis-ci.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# Secrets leak and other sensitive information due to a bug in Travis CI 
+
+Travis CI is a popular continuous integration (CI) and build service,
+originally forked from the Jenkins project, and later spun-off as a paid
+service. On September 23rd, 2021, they reported [1] that users and customers
+of their service, who had sensitive information in public repositories, and
+of which these were forked, had leaked these secrets in the build
+infrastructure logs were the CI service was ran.
+
+
```
Comment (lumjjb) [inline supply-chain-security/compromises/2021/travis-ci.md:10]:
```suggestion
```

linter 

supply-chain-security/compromises/2021/travis-ci.md:10 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]

**Review by lumjjb — COMMENTED** [review]
Thanks @lirantal for the PR, couple nits and should be good to go!

**Comment by lirantal** [comment]
Thanks, going through them now!

File: supply-chain-security/compromises/2021/travis-ci.md (hunk 1 above)
Comment (lirantal) [reply supply-chain-security/compromises/2021/travis-ci.md]: resolved via commit https://github.com/cncf/tag-security/pull/800/commits/3b3a74330adbfc337b44cf8c80eba65ac7fbcffe

**Review by lumjjb — APPROVED** [review]
LGTM! Thanks @lirantal !

**Pull request merged**